src/viam/sdk/robot/client.cpp

```diff
@@ -398,8 +403,7 @@ std::shared_ptr<RobotClient> RobotClient::at_local_socket(std::string address, O
     const char* uri = addr.c_str();
     const std::shared_ptr<grpc::Channel> channel =
         grpc::CreateChannel(uri, grpc::InsecureChannelCredentials());
-    const std::unique_ptr<viam::robot::v1::RobotService::Stub> st =
-        viam::robot::v1::RobotService::NewStub(channel);
+    const std::unique_ptr<RobotService::Stub> st = RobotService::NewStub(channel);
```

Comment:
Is this `st` even used?

Reply:
I guess not. Not sure why it's there. Removed